### PR TITLE
current value turned to default under new variable, for issue #748

### PR DESCRIPTION
**Pull request by new23d**
Proposed code changes to accommodate custom default egress rules, as discussed under issue #748.

Inspired by the GitLab runners code as mentioned under that issue.

Early feedback on these changes is most welcome!

Also, terraform-docs on my end seems to change a lot of the formatting in the README.md files. Am I missing or not doing something obvious? It adds a lot of HTML tags in each line in the tables, hyperlinking to the resources I guess. I'd have thought such formatting should already be present in the current files.

### CODE REVIEW

File: modules/runners/variables.tf [hunk 1]
```diff
@@ -332,3 +332,18 @@ variable "kms_key_arn" {
...
+    cidr_blocks      = list(string)
+    ipv6_cidr_blocks = list(string)
+    prefix_list_ids  = list(string)
+    from_port        = number
+    protocol         = string
+    security_groups  = list(string)
+    self             = bool
+    to_port          = number
+    description      = string
+  }))
```
Comment (npalm) [inline modules/runners/variables.tf:348]: Can you add the same defaults here as the top level variables.tf? This will help the consumers of the submodule.

**Review by npalm — CHANGES_REQUESTED** [review]
Small change requested.

File: modules/runners/variables.tf (hunk 1 above)
Comment (new23d) [reply modules/runners/variables.tf:348]: Done.

**Review by npalm — APPROVED** [review]
@new23d thanks! LGTM

**Pull request merged**